portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DocumentsadministrationUsecase.testcase

```diff
@@ -446,7 +446,7 @@ definition {
 
 		DMDocument.checkoutPG(dmDocumentTitle = "DM Document Title");
 
-		User.logoutUserPG();
+		User.logoutPG();
```

Comment:
I remember there was a logout via url macro but fail to find it, in order to apply the SACRED, do you thinks it's better to logout via url(or other better way to logout the user) to save the user logout time?

Reply:
I talked to Kenji about this but basically there is a more efficient way to login and logout by just controlling the browser cookies. For logout specifically, as long as we are not testing the logout process, we can implement a way to delete the current cookies instead.

Reply:
Thanks for you reply @reizero00, got it, that would save lots of time via sessions. Will wait for it, and try to understand how cookies works with selenium when get a chance.